Model/InitOrderFromQuote.php

```diff
@@ -31,22 +33,33 @@ class InitOrderFromQuote
      */
     private $json;
 
+      /**
+     * @var FlowManager
+     */
+    private $flowManager;
+
     /**
      * @var OrderDataProcessorInterface[]
      */
     private $orderDataProcessors;
 
     /**
-     * @param Flow $flow
+     * @param Config $config
      * @param BoldClient $client
      * @param Json $json
      * @param array $orderDataProcessors
      */
-    public function __construct(Flow $flow, BoldClient $client, Json $json, array $orderDataProcessors = [])
-    {
-        $this->flow = $flow;
```

Comment:
[Discussion] I moved the flow model as well and opted for a FlowManager type file. I'm not sure if Magento has specific preferences on the folder structure or if a FlowService naming it more suited. 
Open to opinions on this.

Reply:
This feels like a Service to me, but I'm also open to following any precedent that exists for the project.
Side note, the `getOrCreate()` function could also be moved here.

Reply:
I agree with Eric, and I probably would've used one here.

Reply:
Yep makes sense.
Renamed it here to be a service: be523967c4f6071d4a967e71573bcf98d236f017